DEVELOPING.md

```diff
@@ -1,30 +1,99 @@
-# Developing the Native Module using Rust
+# Developing the Fluvio Node.js Client
 
-For those who would like to build the native module, follow these instructions:
-
-## Install Rust tooling
-
-First, install rust tooling by following [rustup](https://rustup.rs).
+## Module Hierarchy
 
 ```
-<follow instruction on rustup>
+- @fluvio/client
+    - @fluvio/native
+        - @fluvio/native-darwin
+        - @fluvio/native-linux
+        - @fluvio/native-win
 ```
 
-Install build tool
-```
-cargo install nj-cli
-```
+## Developing the Native Platform Specific Modules
+
+### Pre-Requisites
 
-# Development Build and Test
+- [`Rust`](https://rustup.rs)
+- [`Node-Bindgen`](https://github.com/infinyon/node-bindgen)
 
-To compile rust library:
+Building the `@fluvio/native-<platform>` native modules requires the `Rust` and `Node-Bindgen`. 
+
+Follow the instructions in the links above to install the development tools.
+
+## Building and Testing
+
+### Developing the Native Module
+
+Running `make build` will build the `./src/**/*.{ts,rs}` files, generating a `dist/` folder for the JavaScript files and a native module at the location `./native/src/<platform>/index.node`.
```

Comment:
This should be only required for non supported platform right?

Reply:
This would be required for developing the native module for any platform.